docs/src/main/sphinx/release/release-448.md

```diff
@@ -0,0 +1,48 @@
+# Release 448 (15 May 2024)
+
+## General
+
+* Fix query failure when reading columns with names that are entirely
+  non-alphanumeric. ({issue}`21833`)
+* Fix potential incorrect results for queries with complex predicates. ({issue}`21887`)
+* Fix potential query hang when multiple queries are failing at the same time. ({issue}`21896`)
+* Fix startup failure when fault-tolerant execution is enabled with Google
+  Cloud Storage exchange. ({issue}`21951`)
+* Fix potential failure when queries contain `try_cast`. ({issue}`21952`)
+* Fix graceful shutdown potentially hanging indefinitely when a worker node has
+  crashed. ({issue}`18329`)
+
+## Delta Lake connector
+
+* Add support for caching Glue metadata. ({issue}`20657`)
+* Update Glue to V2 REST interface. The old implementation can be temporarily
+  restored by setting the `hive.metastore` configuration property to `glue-v1`. ({issue}`20657`)
+* Improve performance of reading from Parquet files on platforms with at least
```

Comment:
I believe there are improvements even in the case those instructions are not available. I would remove the latter part.